### PR TITLE
Video

### PR DESCRIPTION
**Pull request by vfarcic**
Added a video to the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Follow the setup recommendations of your cloud provider.
 - [Contributing](CONTRIBUTING.md)
 
 ## Talks
+- [How To Auto-Scale Kubernetes Clusters With Karpenter](https://youtu.be/C-2v7HT-uSA)
```
Comment (ellistarn) [inline README.md:47]: This is getting a little bit hard to parse without some organization/categorization. 

Can you put a date on this and the other ones, e.g. `[12/20/21] How To Auto-Scale Kubernetes Clusters With Karpenter`

Do you mind back

**Review by ellistarn — COMMENTED** [review]
Great video, thank you!

**Comment by vfarcic** [comment]
Something like this @ellistarn?

**Review by ellistarn — COMMENTED** [review]
Great, thank you!

**Review by ellistarn — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -44,10 +44,11 @@ Follow the setup recommendations of your cloud provider.
 - [Contributing](CONTRIBUTING.md)
 
 ## Talks
-- [Karpenter vs Kubernetes Cluster Autoscaler](https://youtu.be/3QsVRHVdOnM)
-- [Karpenter @ Container Day, October 2021](https://youtu.be/qxWJRUF6JJc)
-- [Karpenter @ Container Day, May 2021](https://youtu.be/MZ-4HzOC_ac?t=7137)
-- [Groupless Autoscaling with Karpenter @ Kubecon, May 2021](https://www.youtube.com/watch?v=43g8uPohTgc)
+- 12/20/21 [How To Auto-Scale Kubernetes Clusters With Karpenter](https://youtu.be/C-2v7HT-uSA)
```
Comment (ellistarn) [inline README.md:47]:
```suggestion
- 12/20/2021 [How To Auto-Scale Kubernetes Clusters With Karpenter](https://youtu.be/C-2v7HT-uSA)
```

**Review by ellistarn — APPROVED** [review]
